extensions/wikia/Email/SocialLinksGenerator.class.php

```diff
@@ -0,0 +1,187 @@
+				'footerIcon' => 'Twitter-Icon-2x.png',
+				'welcomeIcon' => 'Connect-Tw.png',
+			],
+			'Instagram' => [
+				'url' => 'https://www.instagram.com/de_fandom',
+				'footerIcon' => 'Instagram-Default-2x.png',
+				'welcomeIcon' => 'Connect-IG.png',
+			],
+		],
+		'it' => [
+			'Facebook' => [
+				'url' => 'https://www.facebook.com/fandom.italy',
+				'footerIcon' => 'Facebook-Icon-2x.png',
+				'welcomeIcon' => 'Connect-FB.png',
+			],
+			'Twitter' => [
+				'url' => 'https://twitter.com/fandom_italy',
+				'footerIcon' => 'Twitter-Icon-2x.png',
+				'welcomeIcon' => 'Connect-Tw.png',
+			],
+		],
+		'ja' => [
+			'Facebook' => [
+				'url' => 'https://www.facebook.com/FandomJP',
+				'footerIcon' => 'Facebook-Icon-2x.png',
+				'welcomeIcon' => 'Connect-FB.png',
+			],
+			'Twitter' => [
+				'url' => 'https://twitter.com/FandomJP',
+				'footerIcon' => 'Twitter-Icon-2x.png',
+				'welcomeIcon' => 'Connect-Tw.png',
+			],
+		],
+		'pl' => [
+			'Facebook' => [
+				'url' => 'https://www.facebook.com/pl.fandom',
+				'footerIcon' => 'Facebook-Icon-2x.png',
+				'welcomeIcon' => 'Connect-FB.png',
+			],
+			'Twitter' => [
+				'url' => 'https://twitter.com/pl_fandom',
+				'footerIcon' => 'Twitter-Icon-2x.png',
+				'welcomeIcon' => 'Connect-Tw.png',
+			],
+		],
+		'br' => [
+			'Facebook' => [
+				'url' => 'https://www.facebook.com/getfandom.br',
+				'footerIcon' => 'Facebook-Icon-2x.png',
+				'welcomeIcon' => 'Connect-FB.png',
+			],
+			'Twitter' => [
+				'url' => 'https://twitter.com/getfandom_br',
+				'footerIcon' => 'Twitter-Icon-2x.png',
+				'welcomeIcon' => 'Connect-Tw.png',
+			],
+		],
+		'ru' => [
+			'Facebook' => [
+				'url' => 'https://www.facebook.com/ru.fandom',
+				'footerIcon' => 'Facebook-Icon-2x.png',
+				'welcomeIcon' => 'Connect-FB.png',
+			],
+			'Twitter' => [
+				'url' => 'https://twitter.com/ru_fandom',
+				'footerIcon' => 'Twitter-Icon-2x.png',
+				'welcomeIcon' => 'Connect-Tw.png',
+			],
+			'VK' => [
+				'url' => 'https://vk.com/ru_fandom',
+				'footerIcon' => 'VK-Icon.png',
+				'welcomeIcon' => 'Connect-VK.png',
+			],
+		],
+		'es' => [
+			'Facebook' => [
+				'url' => 'https://www.facebook.com/Fandom.espanol',
+				'footerIcon' => 'Facebook-Icon-2x.png',
+				'welcomeIcon' => 'Connect-FB.png',
+			],
+			'Twitter' => [
+				'url' => 'https://twitter.com/es_fandom',
+				'footerIcon' => 'Twitter-Icon-2x.png',
+				'welcomeIcon' => 'Connect-Tw.png',
+			],
+			'Instagram' => [
+				'url' => 'https://www.instagram.com/es_fandom',
+				'footerIcon' => 'Instagram-Default-2x.png',
+				'welcomeIcon' => 'Connect-IG.png',
+			],
+		],
+	];
+
+	/**
+	 * Generates links for social icons.
+	 *
+	 * @param $lang - language string
+	 * @param $iconKey - key from which icon file name should be taken
+	 * @return array - returns array of icons
+	 */
+	public static function generate($lang, $iconKey = 'footerIcon') {
+		$socials = self::SOCIAL_MEDIA;
+		if(!isset($socials[$lang])) {
```

Comment:
Unfortunately, `isset` function works only with variable, but I'll change it to `array_key_exists` function.